crates/conjure_core/src/rules/minion.rs

```diff
@@ -290,10 +362,26 @@ fn introduce_weighted_sumleq_sumgeq(expr: &Expr, model: &Model) -> ApplicationRe
         return Err(RuleNotApplicable);
```

Comment:
What if here, instead of saying not applicable we went ahead and produced a non-weighted sumgeq/sumleq? Would this allow us to remove the rule that introduces the non-weighted version?

It's just this rule does a lot of work and if all coefficients were indeed 1, it is clear what we would want to do.